content/en/docs/Writing policies/generate.md

```diff
@@ -23,7 +23,7 @@ Deleting the policy containing a `generate` rule with a `data` object and `synch
 Kubernetes has many default resource types even before considering CustomResources defined in CustomResourceDefinitions (CRDs). While Kyverno can generate these CustomResources as well, both these as well as certain default Kubernetes resources may require granting additional privileges to the ClusterRole responsible for the `generate` behavior. To enable Kyverno to generate these other types, edit the ClusterRole typically named `kyverno:generatecontroller` and add or update the rules to cover the resources and verbs needed.
 
 {{% alert title="Note" color="info" %}}
-For generating a custom resource using generate policy, it is necessary to mention the resource `apiVersion` and `kind`.
+For generating a custom resource using generate policy, it is necessary to set the resource apiVersion i.e `cpol.generate.apiVersion` and kind i.e `cpol.generate.kind`.
```

Comment:
Is it `cpol.generate.apiVersion` or `cpol.generate.data.metadata.apiVersion`?

Reply:
I'd suggest `spec.generate.apiVersion` and not the alias for the CustomResource itself.

Reply:
Suggested: "When generating a custom resource, it is necessary to set the apiVersion (ex., `generate.apiVersion` and kind (ex., `generate.kind`)."